CONTRIBUTING.md

```diff
@@ -37,14 +37,38 @@ Here is an [Example](https://github.com/cucumber/bool/pull/12) of this process i
 2. Close tickets with commits if you can
     + Add "Closes #5, #9" somewhere in the commit message to both link and close. See [Issues 2.0 the Next Generation](https://github.com/blog/831-issues-2-0-the-next-generation) for details.
     + Use [this script](https://gist.github.com/aslakhellesoy/4754009) to compile and view GFM locally
-3. Tag issues so we can do better triage and assignment.
-    + People tend to gravitate towards areas of expertise and tags makes it easier to give a ticket to the right person.
-4. Update HISTORY.md
-    + When you fix a bug or add a feature
-    + Add release dates
-5. Subscribe to ticket feeds so you stay in the loop and get a chance to provide feedback on tickets
-6. The code standard is the existing code
+3. Subscribe to ticket feeds so you stay in the loop and get a chance to provide feedback on tickets
+4. The code standard is the existing code
     + Use the same indentation, spacing, line ending, ASCII for source code, UTF-8 everywhere else
-7. Use git diff (or git diff --cached if you have staged) before every commit
+5. Use git diff (or git diff --cached if you have staged) before every commit
     + This helps you avoid committing changes you didn't mean to
 
+## Maintainers' guide
+
+### Merge a PR
+
+- Verify that:
+  - All checks have passed
+  - At least one maintainer has approved any non-trivial change, and a
+    discussion has occurred for any breaking change
+  - The branch does not need rebasing or squashing of commits
+- To merge:
+  - Follow the command line instructions on GitHub
+  - If it is either a new feature or a bugfix, specify the `--no-commit`
+    flag when merging and add a line to `HISTORY.md` following the
+    current convention. Add this file to the changes to be committed and
+    commit the merge.
+  - Commit message should follow the current convention:
+    `Merge #42 'Description of the change usually from the PR description'`
```

Comment:
Is this the message github auto generates when you use the web interface for merging? If so then I think it is helpful to mention that fact here.